models/access_test.go

```diff
@@ -5,8 +5,8 @@
 package models
 
 import (
-	"testing"
 	"github.com/stretchr/testify/assert"
```

Comment:
You must be following import path rule like https://github.com/go-gitea/gitea/blob/master/main.go#L9-L16